Consul WAN Fed with Vault Secrets Backend document updates (#16597)
hashicorp/consul: website/content/docs/k8s/deployment-configurations/vault/wan-federation.mdx

## Next steps
You have completed the process of federating the secondary datacenter (dc2) with the primary datacenter (dc1) using Vault as the Secrets backend.  To validate that everything is configured properly, please confirm that all pods within both datacenters are in a running state.

> For further detail on specific Consul secrets that are available to be stored in Vault, please check out the detailed information in the [Data Integration](/consul/docs/k8s/deployment-configurations/vault/data-integration) section of the [Vault as a Secrets Backend](/consul/docs/k8s/deployment-configurations/vault) area of the Consul on Kubernetes documentation.

Reviewer: ```suggestion
For additional information about specific Consul secrets that you can store in Vault, refer to [Data Integration](/consul/docs/k8s/deployment-configurations/vault/data-integration) in the [Vault as a Secrets Backend](/consul/docs/k8s/deployment-configurations/vault) documentation.
```
This is a little cleaner and more consistent with our writing guidelines.
